source/plugin-changelogs.txt

```diff
@@ -0,0 +1,41 @@
+.. _atlas-cli-plugin-changelogs:
+
+:noprevnext:
+
+=======================
+{+atlas-cli+} Changelog
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _ak8so-cli-plugin-changelog:
+
+{+ak8so+} CLI Plugin
+----------------------------------------------------
+
+.. _ak8so-cli-plugin-1.1.1-changelog:
+
+{+ak8so+} CLI Plugin 1.1.1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Updates the :ref:`atlas-kubernetes-config-apply` command to support the same features as
+  the :ref:`atlas-kubernetes-config-generate` command.
+
+  :ref:`atlas-kubernetes-config-apply` now supports the following flags, which were
+  introduced to :ref:`atlas-kubernetes-config-generate` in previous releases:
+
+  - ``--dataFederationName``
+  - ``--independentResources``
+
+This release of the {+ak8so+} CLI Plugin is compatible with the 
+following {+ak8so+} releases:
+
+- 2.8.0
```

Comment:
wasn't sure if N-2 was major or patch releases. Went with major for now.

Reply:
the list here is correct 👍